Assets/BossRoom/Scripts/Shared/Game/UI/Editor/NetworkLatencyWarning.cs

```diff
@@ -25,19 +27,20 @@ void Update()
 
                 switch (chosenTransport)
                 {
-                    // adding this preprocessor directive check since LiteNetLib only injects latency in #DEBUG
-                    // todo MTT-1426 do this for UTP
-                    // #if DEBUG
-                    // case LiteNetLibTransport liteNetLibTransport:
-                    //     m_ArtificialLatencyEnabled = liteNetLibTransport.SimulatePacketLossChance > 0 ||
-                    //         liteNetLibTransport.SimulateMinLatency > 0 ||
-                    //         liteNetLibTransport.SimulateMaxLatency > 0;
-                    //     break;
-                    // #endif
                     case UNetTransport unetTransport:
+                        break;
```

Comment:
what if you use artificial latency in UTP and then you start a new match using photon (in the same game run)?

Reply:
Nice catch, I didn't know we could do that! I'll remove the warning for those cases

Reply:
I believe that's why there was no break in the original code for each case

Reply:
instead of setting to false in each case, you could just revert it to what it was